scripts/manage-db.py

```diff
@@ -84,13 +85,57 @@ def cleanup_releases_history(trans, dryrun=True):
 
     print "Total Deleted: %d" % total_deleted
 
+
+def extract_active_data(URL, loc="dump.sql"):
+    """
+    Function was added in to enhance testing in Balrog's stage environment.
+
+    :param URL: Database. eg : mysql://balrogadmin:balrogadmin@balrogdb/balrog
+    :param loc: location where sqldump file must be created
+    :return: Stores sqldump data in  the specified location, if not specified stores in current directory in file dump.sql
+            If  file already exists it will override that file and not append it.
+    """
+    url = make_url(URL)
+    user = url.username
+    password = url.password
+    host = url.host
+    database = url.database
+
+    popen('mysqldump -h %s -u %s -p%s --single-transaction %s rules rules_history rules_scheduled_changes \
+           rules_scheduled_changes_history permissions permissions_history migrate_version  > %s' % (host, user, password, database, loc,))
+
+    popen('mysqldump -h %s -u %s -p%s --single-transaction %s releases \
+           --where="exists (select * from rules where releases.name = rules.mapping  )" \
+           >> %s' % (host, user, password, database, loc,))
+
+    popen('mysqldump -h %s -u %s -p%s --single-transaction %s releases \
+           --where="exists (select * from rules where releases.name = rules.whitelist  )" \
+               >> %s ' % (host, user, password, database, loc,))
+
+    popen('mysqldump -h %s -u %s -p%s --single-transaction %s releases \
+          --where="exists (select * from rules_scheduled_changes where releases.name = rules_scheduled_changes.base_mapping  )"\
+            >> %s' % (host, user, password, database, loc,))
+
+    popen('mysqldump -h %s -u %s -p%s --single-transaction %s releases \
+          --where="exists (select * from rules_scheduled_changes where releases.name = rules_scheduled_changes.base_whitelist )"\
+                >> %s' % (host, user, password, database, loc,))
```

Comment:
I think we're going to run into some issues with duplicate rows if these are run in separate queries. It's possible that the same releases.name row will match multiple queries (in particular, the 1st and the 3rd). Can you combine this into one larger query to avoid that problem?